<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReportViewer</class>
 <widget class="QWidget" name="ReportViewer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>733</width>
    <height>513</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pageSetup">
       <property name="text">
        <string>Page Setup</string>
       </property>
       <property name="icon">
        <iconset resource="rc.qrc">
         <normaloff>:/images/page-setup-32.png</normaloff>:/images/page-setup-32.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>16</width>
         <height>16</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="print">
       <property name="text">
        <string>Print</string>
       </property>
       <property name="icon">
        <iconset resource="rc.qrc">
         <normaloff>:/images/print-32.png</normaloff>:/images/print-32.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>16</width>
         <height>16</height>
        </size>
       </property>
       <property name="shortcut">
        <string>Ctrl+P</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="savePDF">
       <property name="text">
        <string>Save PDF</string>
       </property>
       <property name="iconSize">
        <size>
         <width>16</width>
         <height>16</height>
        </size>
       </property>
       <property name="shortcut">
        <string>Ctrl+S</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="FirstPage">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="rc.qrc">
         <normaloff>:/images/go-first-32.png</normaloff>:/images/go-first-32.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>16</width>
         <height>16</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="PreviousPage">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="rc.qrc">
         <normaloff>:/images/go-previous-32.png</normaloff>:/images/go-previous-32.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>16</width>
         <height>16</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="PageLabel">
       <property name="text">
        <string>Page 0/0</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="NextPage">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="rc.qrc">
         <normaloff>:/images/go-next-32.png</normaloff>:/images/go-next-32.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>16</width>
         <height>16</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="LastPage">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="rc.qrc">
         <normaloff>:/images/go-last-32.png</normaloff>:/images/go-last-32.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>16</width>
         <height>16</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QToolButton" name="zoomIn">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="rc.qrc">
         <normaloff>:/images/zoom-in-32.png</normaloff>:/images/zoom-in-32.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>16</width>
         <height>16</height>
        </size>
       </property>
       <property name="shortcut">
        <string>Ctrl++</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="zoomOut">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="rc.qrc">
         <normaloff>:/images/zoom-out-32.png</normaloff>:/images/zoom-out-32.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>16</width>
         <height>16</height>
        </size>
       </property>
       <property name="shortcut">
        <string>Ctrl+-</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPrintPreviewWidget" name="widget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>2</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QPrintPreviewWidget</class>
   <extends>QWidget</extends>
   <header>qprintpreviewwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="rc.qrc"/>
 </resources>
 <connections/>
</ui>
